provider/ec2/local_test.go

```diff
@@ -2410,18 +2413,26 @@ func (t *localServerSuite) TestStopInstances(c *gc.C) {
 	// for Instances to return an error, and it will not retry
 	// if it succeeds.
 	gone := false
-	for a := ec2.ShortAttempt.Start(); a.Next(); {
+	retryStrategy := retry.CallArgs{
```

Comment:
This can use `ShortRetryStrategy` from export_test.go right?

Reply:
Done